code/game/objects/items/weapons/mop.dm

```diff
@@ -26,7 +26,7 @@ obj/item/weapon/mop/proc/clean(turf/simulated/A)
 		if(istype(F))
 			F.dirt = 0
 		for(var/obj/effect/O in A)
-			if(istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
+			if(istype(O,/obj/effect/decal/cleanable))

```

Comment:
Do you mean for example a global var(or a var for /obj), like able_to_clean, and then adding it to each thing which can be cleaned, and calling that instead of /obj/effect/decal/cleanable?